common.c

```diff
@@ -30,6 +32,31 @@
 static void log_error(const char *fmt, ...)
 	__attribute__ ((format (printf, 1, 2)));
 
+
+static const char* get_curtime_string()
+{
+	// This will invalidate previous return pointer on the next call.
+	// Doesn't really matter for us, since we're using it immediately.
+	static char buffer[40];
+
+	// https://stackoverflow.com/a/32983646
+	struct timeval tv;
+	gettimeofday(&tv, NULL);
+
+	int millisec = tv.tv_usec/1000;
+	if (millisec>=1000) { // Allow for rounding up to nearest second
+		millisec -=1000;
+		tv.tv_sec++;
+	}
+
+	struct tm* tm_info = localtime(&tv.tv_sec);
+
+	// https://stackoverflow.com/a/2023961
+	size_t len = strftime(buffer, 26, "%Y:%m:%d %H:%M:%S", tm_info);
```

Comment:
I'm wondering if it at the highest log levels this could slow down logging. Have you noticed anything like that / do you think it would make sense to make this an option?

Reply:
`2024:06:24` is a strange date format, did you mean to write `%Y-%m-%d`?

Reply:
> I'm wondering if it at the highest log levels this could slow down logging

I do notice a little bit of slowdown, but i'm logging to tmpfs, and my logs are >150MiB. I don't think this is too much of an issue.

> do you think it would make sense to make this an option?

I did think about it, but then i'd have to reorder code, or extern stuff, and also make sure that config loading happens before reading config.

> 2024:06:24 is a strange date format, did you mean to write %Y-%m-%d?

I agree.

Reply:
> I did think about it, but then i'd have to reorder code, or extern stuff, and also make sure that config loading happens before reading config.

The `Debug` setting is in the configuration, so it is read before any logging happens. Is this different?

Reply:
It was just me being lazy. I'll see what i can do.